### PR TITLE
Bug 1746716: grafana: Set default admin user to impossible openshift user

### PR DESCRIPTION
**Pull request by paulfantom**
Backport https://github.com/openshift/cluster-monitoring-operator/pull/361

/cc @s-urbaniak

### CODE REVIEW

**Comment by LiliC** [comment]
/retest

seems like something got stuck

**Comment by LiliC** [comment]
/hold

Think you need to rebase, as generate is failing.

File: Makefile [hunk 1]
```diff
@@ -1,110 +1,154 @@
...
+##############
+# Generating #
+##############
 
-clean:
-	rm -f $(BIN)
-	go clean -r $(MAIN_PKG)
-	docker images -q $(REPO) | xargs --no-run-if-empty docker rmi --force
-	rm -rf jsonnet/vendor
+.PHONY: vendor
```
Comment (LiliC) [inline Makefile]: I am guessing we don't call this ever, I am fine with having this for consistency with other versions. But we still don't use gomods correctly?

**Comment by paulfantom** [comment]
manual CI jobs are now synced across branches, this means we need to provide same scripts and same execution methods for those. In our case `ci/prow/generate` and `ci/prow/shellcheck` are the "manual" jobs.
In 4.1 we weren't running shellcheck, but job needs anything to be present in `hack/shellcheck.sh`. Since script from 4.2 and onwards fail on 4.1 I decided to do a workaround of `exit 0`.
As for `ci/prow/generate` I backported Makefile from 4.2 and disabled go modules in it. Now we run the same generation step with jsonnet-ci image.

File: Makefile (hunk 1 above)
Comment (paulfantom) [reply Makefile]: Argh, we need to disable go mods globaly here as we weren't using them in 4.1.

**Comment by ashcrow** [comment]
Is this expected?

```
$ make dependencies build

make: *** No rule to make target 'dependencies'.  Stop.

The command "make dependencies build" exited with 2.
```

**Comment by LiliC** [comment]
/hold

Until Pawel comes back from PTO or someone takes over this. IT seems the travis build is complaining about what @ashcrow mentioned.

**Comment by bparees** [comment]
This looks potentially important enough to go into 4.1.z(it's a security fix) but it's failing and is held, so it's not being cherry-picked-approved at this time.

**Comment by paulfantom** [comment]
This PR also obsoletes travis CI tests as everything is covered by prow.

**Comment by s-urbaniak** [comment]
/lgtm
as discussed out of band. There is nothing in Travis any more that is not covered via prow for 4.1.

**Pull request merged**